src/components/Product.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function Product(props) {
+  const {product} = props;
```

Comment:
You can define your object directly in the component like this:
```suggestion
export default function Product({product}) {
```